.github/workflows/go.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     - name: Set up Go 1.18
       uses: actions/setup-go@v3
       with:
-        go-version: '>=1.18.2'
+        go-version: 1.18.2
```

Comment:
It's better to specify '<1.19' until we support go v1.19+

Reply:
We also need `>= 1.18.2` for what was discussed on #310. 
Any idea on a clean way to specify both conditions?

Reply:
I think we can go with <1.19 there is already 1.18.5 so there is no risk here

Reply:
ok